.github/workflows/tests.yml

```diff
@@ -0,0 +1,41 @@
+# Inspired by:
+# https://github.com/zopefoundation/meta/tree/master/config/pure-python
+name: tests
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+  schedule:
+    - cron: "3 14 * * 6" # run once a week on Saturday
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [2.7, 3.6, 3.7, 3.8]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Pip cache
+        uses: actions/cache@v2
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('setup.*', '*.cfg') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install -r requirements.txt
+          sudo apt-get install libxml2-dev libxslt-dev xvfb
```

Comment:
Why are these apt dependencies needed?
I can imagine the first two, for compiling lxml, but then I wonder why they are not needed by other Plone projects. Or are they?
`xvfb` should not be needed: we do not need an X server for running a browser.

Reply:
I saw the build failing like this:
```
ERROR: b'/bin/sh: 1: xslt-config: not found\n'
** make sure the development packages of libxml2 and libxslt are installed **
```
See https://github.com/collective/collective.recipe.backup/runs/2585100630?check_suite_focus=true
Then I copied-pasted that line from another GHA actions wf.
Obviously `xvfb` is not needed. I will remove it.

Reply:
I see that in our [buildout config](https://github.com/collective/collective.recipe.backup/blob/master/buildout.cfg) we pin lxml 4.3.3, which has wheels, but we use a somewhat old `zc.buildout` version. Well, 2.13.1 should be able to handle wheels. Anyway, this buildout config can use some love.
We might not need it at all. Well, making sure this recipe runs in an actual buildout is good. Anyway, something for another day.